safety-analysis/telltale/README.md

```diff
@@ -0,0 +1,21 @@
+# Telltale Safety Analysis
+
+## Purpose and scope
+
+This section of the repository the OSEP working group's analysis of the Automotive Working Group's [Instrument Cluster use case](https://github.com/elisa-tech/wg-automotive/tree/master/Cluster_Display_Use_Case_v2).
```

Comment:
missing a verb - suggest '[...] repository hosts the [...]'

Reply:
Fixed